src/ansi-c/library/math.c

```diff
@@ -1382,11 +1262,9 @@ long double __sort_of_CPROVER_round_to_integrall (int rounding_mode, long double
 #define __CPROVER_FENV_H_INCLUDED
 #endif
 
-double __sort_of_CPROVER_round_to_integral (int rounding_mode, double d);
-
 double ceil(double x)
 {
-  return __sort_of_CPROVER_round_to_integral(FE_UPWARD, x);
+  return __CPROVER_round_to_integrald(x, 2); // FE_UPWARD
```

Comment:
Why did you choose not to use the named constant `FE_UPWARD`?

Reply:
`__CPROVER_round_to_integrald` uses our own numbering, which may or may not match `FE_UPWARD` on the target platform.